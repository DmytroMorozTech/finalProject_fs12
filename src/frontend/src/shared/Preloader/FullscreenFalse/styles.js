import { makeStyles } from '@material-ui/core/styles'

export default makeStyles(() => ({

  fullscreenFalse: {
    display: 'flex',
    justifyContent: 'center',
    height: '100%',
    width: '100%',
    margin: '10px auto'
  }
}))
