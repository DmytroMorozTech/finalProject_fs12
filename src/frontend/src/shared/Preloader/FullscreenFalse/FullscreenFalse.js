import { CircularProgress } from '@material-ui/core'
import styles from './styles'
import React from 'react'

function FullscreenFalse () {
  const classes = styles()

  return (
    <div className={classes.fullscreenFalse}>
      <CircularProgress/>
    </div>
  )
}

export default FullscreenFalse
