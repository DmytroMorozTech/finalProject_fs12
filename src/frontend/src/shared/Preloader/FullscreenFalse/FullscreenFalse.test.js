import React from 'react'
import FullscreenFalse from './FullscreenFalse'
import { shallow } from 'enzyme'

const setUp = (props) => shallow(<FullscreenFalse {...props}/>)

describe('should render FullscreenFalse component', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
