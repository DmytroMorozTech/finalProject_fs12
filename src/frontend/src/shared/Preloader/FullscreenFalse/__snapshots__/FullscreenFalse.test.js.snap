// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FullscreenFalse component to match snapshot 1`] = `
<div
  className="makeStyles-fullscreenFalse-1"
>
  <WithStyles(ForwardRef(CircularProgress)) />
</div>
`;
